api/config/v1alpha1/defaults.go

```diff
@@ -129,6 +129,8 @@ const (
 	DefaultCompactionConcurrentSyncs = 3
 	// DefaultCompactionEventsThreshold is the default number of events that triggers compaction.
 	DefaultCompactionEventsThreshold = 1000000
+	// DefaultCompactionTriggerFullSnapshotThreshold is the default upper threshold for the number of etcd revisions before giving up on compaction job and triggering a full snapshot.
```

Comment:
```suggestion
	// DefaultCompactionTriggerFullSnapshotThreshold is the default upper threshold for the number of etcd events before giving up on compaction job and triggering a full snapshot.
```

Reply:
be consistent and used events or revisions. For `DefaultCompactionEventsThreshold` we use events. If we are going to also measure thresholds via number of events then we should adjust the doc string and also rename this variable to `DefaultCompactionTriggerFullSnapshotEventsThreshold`